students/zach_thomson/lesson03/mailroom.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+import sys
+
+donor_db = [('Eddie Vedder', [10000.00, 20000.00, 4500.00]),
+            ('Chris Cornell', [100.00, 500.00]),
+            ('Kurt Cobain', [25.00]),
+            ('Dave Matthews', [100000.00, 50000.00, 125000.00]),
+            ('Dave Grohl', [50.00])]
+
+
+prompt = '\n'.join(('Welcome to the mailroom',
+         'Please choose from the following options:',
+         '1 - Send a Thank You',
+         '2 - Create a Report',
+         '3 - Quit',
+         '> '))
+
+#send a thank you tasks
+def donor_in_list(x):
+    """takes in a name and determines if it is in the donor database"""
+    i = 0
+    result = False
+    while i < len(donor_db):
+        if x == donor_db[i][0]:
+            result = True
+            break
+        else:
+            i = i + 1
+    return result
+
+def donor_idx(x):
+    """takes in a name and finds the index number in donor database"""
+    i = 0
+    while i < len(donor_db):
+        if x == donor_db[i][0]:
+            return i
+        else:
+            i += 1
+
+def donor_names():
+    """return a list of names in the donor database"""
+    i = 0
+    donors = []
+    while i < len(donor_db):
+        donors.append(donor_db[i][0])
+        i += 1
+    return donors
+
+donation_email = "\nDear {},\nThank you for your generous donation of ${:.2f}!\n"
+
+def thank_you():
+    ty_prompt = input('Please enter a full name or type list to see all the current donors: ')
+    while ty_prompt.lower() == 'list':
+        print(donor_names())
+        ty_prompt = input('Please enter a full name or type list to see all the current donors: ')
+    else:
+        if donor_in_list(ty_prompt) == True:
```

Comment:
use `is` with objects instead of `==`

also, you can just have this as `if donor_in_list(ty_prompt):`